package/MDAnalysis/coordinates/DCD.py

```diff
@@ -550,106 +307,123 @@ def timeseries(self, asel=None, start=None, stop=None, step=None, skip=None,
+            number of atoms to be written
+        convert_units : bool (optional)
+            convert from MDAnalysis units to format specific units
         step : int (optional)
-            indicate that `step` **MD steps (!)** make up one trajectory frame
-        delta : float (optional)
-            **MD integrator time step (!)**, in AKMA units
+            number of steps between frames to be written
         dt : float (optional)
-            **Override** `step` and `delta` so that the DCD records that
-            `dt` ps lie between two frames. (It sets `step` `` = 1`` and
-            `delta` `` = AKMA(dt)``.) The default is ``None``, in which case
-            `step` and `delta` are used.
-
+            use this time step in DCD. If ``None`` guess from first written
+            TimeStep
         remarks : str (optional)
-            string that is stored in the DCD header
-
-        Returns
-        -------
-        :class:`DCDWriter`
-
-
-        Note
-        ----
-        The keyword arguments set the low-level attributes of the DCD according
-        to the CHARMM format. The time between two frames would be `delta` *
-        `step`!
-
-        Here `step` is really the number of MD integrator time steps that
-        occured after this frame, including the frame itself that is the
-        coordinate snapshot and `delta` is the integrator stime step.  The DCD
-        file format contains this information so it needs to be provided here.
+            remarks to be stored in DCD. Shouldn't be more then 240 characters
+        nsavc : int (optional)
+            DCD usually saves ``dt`` as the integrator timestep and the
+            frequency of writes in the nsavc variable separately. Unless you
+            know what you are doing you don't need to touch this value.
+            If you plan to use the written DCD with another tool that depends
+            on this behavior you can adjust it with this variable. The DCD
+            reader will then interpret the timestep between frames as ``dt *
+            nsavc``.
+        **kwargs : dict
+            General writer arguments
+        """
+        self.filename = filename
+        self._convert_units = convert_units
+        if n_atoms is None:
+            raise ValueError("n_atoms argument is required")
+        self.n_atoms = n_atoms
+        self._file = DCDFile(self.filename, 'w')
+        self.step = step
+        self.dt = dt
+        dt = mdaunits.convert(dt, 'ps', self.units['time'])
+        self._file.write_header(
+            remarks=remarks,
+            natoms=self.n_atoms,
+            nsavc=nsavc,
+            delta=float(dt),
+            charmm=1,
+            istart=0)
+
+    def write_next_timestep(self, ts):
+        """Write timestep object into trajectory.
 
+        Parameters
+        ----------
+        ts: TimeStep
 
         See Also
         --------
-        :class:`DCDWriter`
-
+        <FormatWriter>.write(AtomGroup/Universe/TimeStep)
```

Comment:
If you want to link to `:meth:'DCDWriter.write'` then add `:inherited-members:` to the reST docs for the `DCDWriter` class and the `write` should show up. (Note, don't copy and paste my reST, I used single quotes instead of backticks because I don't know how to quote characters in markdown.)

Alternatively `:meth:'~MDAnalysis.coordinates.base.BasweWriter.write'` (or similar).